sims/verilator/Makefile

```diff
@@ -1,112 +1,175 @@
-#########################################################################################
+#============================================================================
+# Extra make variables:
+#
+# ENABLE_PRINTF_PATTERN - python regex. any printf() with text that matches
+#                         this regex will be force enabled in sim output
+#                         default is no selectively enabled printf's
+# VL8_PROFILE           - 'none' if no verilator profiling (default)
+#                         'all' if full verilator runtime profiling
+#                         'threads' if runtime thread profiling only
+# VL8_THREADS           - how many threads the simulator will use (default 1)
+# VL8_FST_MODE          - enable FST waveform instead of VCD
+#============================================================================
+
+
+#============================================================================
 # verilator makefile
-#########################################################################################
+#============================================================================
```

Comment:
Why the #/= change?

Reply:
it was more of a 80 char line limit change. no reason for #/= change other than # seems qualitatively less aesthetically pleasing.

Reply:
I think that's subjective (personally I think # is better than = and suspect I'm not alone), but either way stylistic changes should be a separate PR.

Reply:
ok, reverted